Makefile

```diff
@@ -105,4 +108,4 @@ clean: clean-python
 
 .PHONY: fmt tidy vet test-unit test-system test-heavy sampledata test-ci
 .PHONY: perf-compare build install create-release-assets clean clean-python
-.PHONY: build-python-wheel generate test-generate
+.PHONY: build-python-wheel generate test-generate bin/minio
```

Comment:
Do you really want to declare this to be phony?

Reply:
Will this update if go.mod changes to a newer version of bin/minio?